crates/layout/src/layout.rs

```diff
@@ -108,9 +108,10 @@ impl LayoutManager {
     fn update_layouts(&mut self, taffy: &Taffy) -> Vec<i32> {
         fn update_layout(
             layout_id: i32,
+            parent_layout_id: i32,
             manager: &mut LayoutManager,
             taffy: &Taffy,
-            changed: &mut Vec<i32>,
+            changed: &mut HashSet<i32>,
```

Comment:
I'm guessing the Kotlin side doesn't care what order the invalidations are in, so it's OK to move from a Vec to a Set (and avoid duplicate parents...).

Reply:
Yea it doesn't -- it just notifies them all in one pass.